### DISCUSSION TITLE
DOSBox Staging mentions & usages in the wild

### DISCUSSION THREAD

**Discussion by Grounded0** [dosbox-staging/dosbox-staging#3833] [General]
DOS Storm — STOP using DOSBOX! 2 Easier ways to Run DOS Games!
https://www.youtube.com/watch?v=L2KbE0pJbnQ

**Comment by johnnovak** [comment]
PhilsComputerLab — Is DOSBox Staging better than real Retro PC? Part 1 Graphics
https://www.youtube.com/watch?v=zEqinYQ0RtI

PhilsComputerLab — DOSBox Staging 0.81: What can we expect?
https://www.youtube.com/watch?v=ycIw0kDmcEo

**Comment by Grounded0** [comment]
PhilsComputerLab — Is DOSBox Staging better than real Retro PC? Part 2 Audio Sound and Music
https://www.youtube.com/watch?v=2-gVnZbXumQ

**Comment by Grounded0** [comment]
Auto CRT shaders gets a mention:

MetalJesusRocks — Play 7000 PC DOS games quickly and easily (eXoDOS Review)
https://www.youtube.com/watch?v=kFj70j5HNu8

**Comment by johnnovak** [comment]
The guy uses DOSBox Staging as the basis for his "DWEEB_DOS" thingy:

The BEST way to do DOS stuff! // DWEEB_DOS Tour
https://www.youtube.com/watch?v=RdiV9d8uKHI

**Comment by johnnovak** [comment]
eXo explaining the adaptive CRT shaders:

eXoDOS v6: Auto-CRT Shaders
https://www.youtube.com/watch?v=fDyU09r8g5o

**Comment by Grounded0** [comment]
GET OFF MY LAWN — DOS Gaming Is Better Part XX: The Future
https://www.youtube.com/watch?v=wBkT7ZUgg-s

**Comment by johnnovak** [comment]
DOOM + DOOM II ships with DOSBox Staging:

https://steamdb.info/depot/2281/

https://store.steampowered.com/app/2280/DOOM__DOOM_II/

https://www.youtube.com/watch?v=IlPSbp1io7w&t=229s

**Comment by arrowgent** [comment]
The Aching (2023) uses dosbox-staging
https://store.steampowered.com/app/2295210/The_Aching/
https://hadrosoft.itch.io/the-aching
https://www.hadrosaurus.net/the_aching/

**Comment by arrowgent** [comment]
Blood Fresh Supply updated with OUW uses dosbox staging 0.80.1 (2023)
https://steamdb.info/depot/1010753/
https://store.steampowered.com/app/1010750/Blood_Fresh_Supply/
https://store.steampowered.com/news/app/1010750/view/7425934518550758530

**Comment by johnnovak** [comment]
GOG has started shipping **Stonekeep**, **Tex Murphy: Under A Killing Moon**, and **Ultima Underworld 1 & 2** with DOSBox Staging 0.82.

>Switched over to DOSBox Staging version 0.82 for extra features and smoother emulation.

Check out the "What improvements we made to this game" sections:

https://www.gog.com/en/game/stonekeep
https://www.gog.com/en/game/tex_murphy_under_a_killing_moon
https://www.gog.com/en/game/ultima_underworld_1_2

**Reply by MasterO2** [reply]
I wonder if this is the beginning of GOG replacing the original Dosbox in all of their DOS game releases.

**Comment by arrowgent** [comment]
similar to steamdb, gogdb has a good backend for quickly accessing data

https://www.gogdb.org/product/1207658671/releasenotes

though it may not show the files in the packages or easily see which games use the game engine "dosbox"
ah there it is, drill through the manifests, apparently a seperate manifest than EN-US
stored under `*`
https://www.gogdb.org/manifest/7a383e1e7a2cb895aad9d472603cf2e2


ex:
https://steamdb.info/tech/Emulator/DOSBOX/

**Comment by Grounded0** [comment]
GeneralUser GS page specifically mentions DOSBox Staging. Also on the video about GeneralUser GS release 2.0, a game footage is shown running on DOSBox Staging:

https://schristiancollins.com/generaluser.php

https://www.youtube.com/watch?v=AtXvMz22y-M

**Comment by arrowgent** [comment]
new heretic + hexen re-release
uses dosbox-staging 0.81.2 for the dos launchers

**Comment by arrowgent** [comment]
earthworm jim 2 on steam uses dosbox-staging
but not on gog.  gog still using the old dosbox.

**Comment by QmwJlHuSg9pa** [comment]
I've been utilising DOSBox Staging as a basis for my Lutris install scripts (YAML) over the past year or so. My approach has been to identify commonly occurring concerns and develop a sort of informal "best practices" amongst game titles. Essentially, I want the installation experience to be as seamless as possible (minimal busywork for the end user), whilst also "tweaking" the default in-game experience (with respect to the title as an art piece and the conventions of its time).

I'd like to thank the DOSBOX Staging developers/contributors for being approachable and pro-active about the several bug reports I've contributed to, which have allowed me to progress as thus in recent times. Here's a partial summary of my approaches to composing these Lutris install scripts, particularly in the context of utilising DOSBox Staging's modern niceties to their full potential:

### Logistics

- Taking advantage of the "automount" feature, and "drives/" subdirectories.
  - Normalising disc mounts to "imgmount/" subdirectories, in lieu of this feature being implemented into "automount".
- Normalising the main config file with the naming convention "dosbox-NAME.conf".
- Over-stuffing the main config file with environment variable-based IF switches, in order to wrap all launch options within a single file. This is in conjunction with external parameterised options (i.e. directly via Lutris' own launcher GUI).
  E.g. `--set "ipx ipx=true" -c "set RUNSERVER=1" -conf "dosbox-NAME.conf"` -- Explicitly enables networking at runtime (security consideration, as it is otherwise usually disabled within the global .conf file), and signals to enable the `RUNSERVER` routine within the .conf file (which calls "ipxnet startserver", etc.) -- game expansion packs can also be signalled this way, singularly or in plural with other variables (per prior example).
- A bespoke [WBAT](https://www.horstmuc.de/ui.htm#wbat)-based pre-launch GUI, per-title.
  E.g. handling Multiplayer connections via UDP/IPX (as above), as well as handling additional video, audio or control options that would otherwise require tinkering with text files.

### Audio

- Measuring appropriate audio mixer levels (ensuring no clipping via a [K-meter loudness meter](https://github.com/mzuther/K-Meter); compression=off), and considering appropriate reverb presets, per title.
- Taking into consideration the hardware the music was primarily composed on/for (where known), as well providing multiple options, where appropriate (offered via the WBAT GUI, as above).

### Video

- Similar to audio, setting considered new default graphical options (optimal resolution, draw distance, etc. within the scope of developer's gameplay intentions and visual aesthetics) as the out-of-the-box experience, as well as providing further options via WBAT GUI, when notable.

### In-game

- Providing "modern" control sets as optional defaults, going so far as patching the game binaries to acomplish this when creating a new profile in-game.
- Where practical, taking advantage of "autotype" in order to skip pointless launch dialogues; or if selecting a Multiplayer Host/Join option (via Lutris GUI), skipping ahead to the appropriate Multiplayer lobby screen.

Probably more I've forgotten...

Here's an example of such a script: https://lutris.net/games/last-rites/ (unmunged version [last-rites_example.yml.txt](https://github.com/user-attachments/files/21734448/last-rites_example.yml.txt)). I currently have about another 8 or so scripts in a near-ready state.

Here's the basic directory layout of the above game, post-install:

- drives
  - c
    - LASTRITE
    - WBAT
  - u
    - STRING25
    - WBAT250
- imgmount
  - disc1
    - last_rites.iso
- dosbox-LASTRITE.conf

**Comment by johnnovak** [comment]
Very nice @QmwJlHuSg9pa, you're using Staging like a boss! 😎 Glad to see my reverb feature getting some love, and I'm also glad to see there are people out there who are into configuring games in a period-correct way, true to the source material ❤️

**Comment by Grounded0** [comment]
Posting this older one here so we can save space on our open discussions:

Dosbian 2.5 switched from DOSBox-X to DOSBox Staging

https://cmaiolino.wordpress.com/dosbian/

"Added the wonderful Dosbox Staging 0.81"
"Removed Dosbox-X"

It's nice when people give high praise to our releases like this.

**Comment by Grounded0** [comment]
I fear there's more than one machine shop in Germany with CNC machines churning away on instructions provided by DOSBox Staging host at this point. 😅

Greetings to all Indians out there. 🇮🇳

https://snapcraft.io/dosbox-staging

<img width="726" height="428" alt="Screenshot 2025-10-19 at 8 43 13" src="https://github.com/user-attachments/assets/6160fbe8-45b3-492c-bfc3-756261c0fe28" />

**Reply by johnnovak** [reply]
Haha, but that's just Linux. Probably 90% of our users are on Windows — that would paint a very different landscape, potentially.

**Reply by FeralChild64** [reply]
Not even Linux - it’s Ubuntu.

**Comment by Grounded0** [comment]
How-To Geek has been giving a lot of attention to us lately:

DOS Emulation: Which Method Reigns Supreme?

https://www.howtogeek.com/whats-the-most-accurate-way-to-emulate-dos-for-gaming/

5 DOSBox Forks That Are Better Than the Vanilla Release

https://www.howtogeek.com/stop-using-dosbox-and-try-one-of-these-forks-instead/

7 DOSBox Front-Ends That Make Playing Retro PC Games Easy

https://www.howtogeek.com/i-made-dosbox-even-better-with-these-free-front-ends-and-tools/

**Comment by arrowgent** [comment]
blood refreshed supply
steam uses dosbox-staging 0.82.2
`# This is the configuration file for DOSBox Staging (0.82.2). Configured by Saleck.`
https://steamdb.info/depot/3884521/

gog doesnt seem to have the dos included (yet)
https://www.gogdb.org/product/1971203245/build/59216858930725046

**Comment by FeralChild64** [comment]
https://m.youtube.com/watch?v=E1IdWPj34Zw

Using modern GCC for compiling DOS code, uses the 0.83.0-alpha REST API memory viewer.

**Comment by dux119-cmd** [comment]
"Boxer Plus for Mac" announced on Reddit. It's s a wrapper around DOSBox Staging.

https://www.reddit.com/r/macgaming/comments/1smm12u/updated_boxer_app_for_mac_os_supports_both_dos/

**Comment by arrowgent** [comment]
terranova 1996 updated with dosbox-staging

https://store.steampowered.com/news/app/368600/view/535506084665758735

gog also
https://www.gogdb.org/product/1434984562/build/59580639835286995
although no annoucements...

**Comment by dux119-cmd** [comment]
HOW WE GOG? - EPISODE 1 - DOSBOX - GOG Classics Vault and GOG

https://www.youtube.com/watch?v=7-8Y-DRM1Hg

GOG Preservation Program team leader giving thanks to the DOSBox founders QBix and Harekiet:

<img width="1590" height="1057" alt="Screenshot_2026-07-11_16-33-38" src="https://github.com/user-attachments/assets/03e1af36-fae3-4df0-b5c5-289457924935" />

<img width="1590" height="1057" alt="Screenshot_2026-07-11_16-34-07" src="https://github.com/user-attachments/assets/74fe49c1-81be-4cdf-b030-d94d3019d50f" />


Also shows using DOSBox Staging being configured and used to setup games.

I intended to share those screenshots on vogons.org, but those moderators have already banned my account because I answered some forum posts mentioning that DOSBox Staging had fixed some issues being discussed in the thread :( Not sure what's going on there.

**Reply by johnnovak** [reply]
> I intended to share those screenshots on vogons.org, but those moderators have already banned my account because I answered some forum posts mentioning that DOSBox Staging had fixed some issues being discussed in the thread :( Not sure what's going on there.

The original project founder, dreamer, had some unfortunate interactions with the original devs in the VOGONS forum many years ago. Basically he wanted them to adopt modern development standards, they were not receptive to the idea, dreamer got more forceful about it, which resulted in him getting banned from there, and thus the DOSBox Staging project was born.

My personal view is dreamer was more at fault since he tried to force the original DOSBox devs to do things they were not interested in doing.

A rather unfortunate beginning, but that's our history 😅 

Since then, one of the two VOGONS veteran admins (DosFreak and Dominus) still holds a major grudge against us, the other one not so much. From memory, I think Dominus is the reasonable one. I had some personal communication with him, and he said a few appreciative words of the Staging project, but also mentioned the entire undiplomatic conduct of dreamer has kind of "tainted" the entire project for him a little bit. 

I believe they also hold a perception that the Staging project is responsible for some personal issues (including health issues) of one of the original DOSBox devs. I don't know anything about that, but I heard it from some sources that they might be just conflating unrelated health issues with their initial irritation of dreamer or something.

Well, that's the history 😄 Also, they're super sensitive about "abandonware" and "piracy", while we're a lot more relaxed about such things, plus in fairness, Staging doesn't run on older machines on Windows 98 and XP, and most of their userbase is really into retrohardware first, then running DOSBox on their old hardware second. Also, historically VOGONS has been the "official forum" for DOSBox development.

So all in all, we're not welcome there, and it's fine because our interests and approaches don't align anyway. I don't have animosity towards them, I visit their forum from time to time, it's a very good resource for all things retro PC hardware related. I try not to mention Staging there much because usually it doesn't end well for the reasons listed above — it's their forum, we can't "win" there by pushing things.

Now you're up to date 😄

**Reply by dux119-cmd** [reply]
Thanks for the background @johnnovak.

It's too bad they don't mention this rule in their sign up terms: _"The first rule of vogons is to never mention DOSBox Staging."_  (That'd make for a helluva Striesand effect where new members immediately search up DOSBox Staging).

I can understand devs not wanting to do something - I see requests made in plenty of emus, and even when the requestee argues sound technical merits (similar to adopting modern development standards), but of course there are no binding contracts or bully bosses waving threats in the world of open source.

Everyone knows the only "capital" between people is the ability to work constructively with each other leading to motivation (or demotivation) where someone actually wants to spend time (or not) on something.  So it's strange the vogons mods and dosbox guys didn't say, _"Sure, we get the merits, but we're not going to do it because the project is in maintenance-mode. Please fork away and good luck!"_, and just carry on.

> Well, that's the history 😄 Also, they're super sensitive about "abandonware" and "piracy", while we're a lot more relaxed about such things, plus in fairness, Staging doesn't run on older machines on Windows 98 and XP, and most of their userbase is really into retrohardware first, then running DOSBox on their old hardware second. Also, historically VOGONS has been the "official forum" for DOSBox development.

Thanks for describing that separation. I would have got myself in a different bucket of hot water anyway if I brought up using a game from eXoDOS or something like that!

Well, I'm much better off here with the focus being 100% emulation, because I think vogons is mostly a hardware forum with dosbox being mentioned in about 1 in 50 posts, so I won't be trying to create another account on vogons.

**Reply by johnnovak** [reply]
Lol, mentioning eXoDOS even just in passing is the **_best_** way to get banned there 🤣

**Reply by Grounded0** [reply]
It's kinda ridiculous GOG is issuing those special thank you's instead of offering the OG DOSBox or this project any sort of financial support. I've financed this operation for USD 15,000 out of my disability pension and a USD multi-million annual turnover corporation can't afford a USD 1 in financial support seems ridiculous to me.

**Reply by dux119-cmd** [reply]
@Grounded0 - that's super generous! Given GoG financially profited on DOSBox and now increasingly uses and depends on DOSBox Staging, they should be thanking and supporting all their critical open source emulator projects. I know as a user of DOSBox Staging through Batocera, it's brought a ton of platform bug fixes, game fixes, and improvements across the board - input, language, sound, video, and DOS itself (like the share and networking stuff). It's been great!

**Reply by johnnovak** [reply]
I might contact them, but I don't really see the interaction ending differently that getting a "lol, we don't have to pay you, so why would we?" remark. Probably we'd need to offer something in return, like displaying a sponsored by GOG badge and a link on our homepage.

**Reply by Grounded0** [reply]
@johnnovak 

I'm open to corporate sponsors getting their logos and links on the homepage if they're willing to meet the assigned tier monthly cash donation in return. Seems like a good idea.

Here's Mastodon sponsors list as an example how this works:

https://joinmastodon.org/sponsors
https://sponsor.joinmastodon.org

**Reply by arrowgent** [reply]
and so it continues

https://www.youtube.com/watch?v=YKBRbrx2Jas

most of you might already know this one ^^

in purely a retro gaming sphere aspect i've found the dosbox vogons forum interesting historically but very often not useful technically
maybe thats just me glossing over some irrefutable comment that doesnt signal practical application

thats not even me being old and preferring "forums" more than other avenues of discussion boards.
kind of like debating if dosbox should have moved away from sourceforge

much ramblings aside, im going to go watch this $GOG$ video

**Comment by Grounded0** [comment]
DOSBox: How an Emulator Saved PC Gaming History

https://www.technodabbler.com/dosbox-how-an-emulator-saved-pc-gaming-history/

**Comment by Grounded0** [comment]
DOSBox Staging 0.83.0 release coverage:

DOSBOX Staging 0.83 brings authentic CRT colour emulation, FMV deinterlacer, disc automounting and much more

https://www.gamingonlinux.com/2026/08/dosbox-staging-0-83-brings-authentic-crt-colour-emulation-fmv-deinterlacer-disc-automounting-and-much-more/

DOSBox Staging's new update makes DOS gaming better on modern PCs

https://www.generationamiga.com/2026/08/27/dosbox-stagings-new-update-makes-dos-gaming-better-on-modern-pcs/

DOSBox Staging adds authentic Roland SC-55 sound emulation

https://www.retrohrac.cz/en/clanek/dosbox-staging-0-83-0-emuluje-roland-sc-55-d793.html

**Comment by arrowgent** [comment]
with all the hype around dosbox-staging including Sound Canvas Clap and NUKED SC-55

https://github.com/nukeykt/Nuked-SC55/commit/f6e50220d5de4241080e1811a30d29f7113a0929

its now GPL.

good job releasing 0.83.0 btw.  im still working on updating my configs

**Reply by johnnovak** [reply]
> with all the hype around dosbox-staging including Sound Canvas Clap and NUKED SC-55
> 
> [nukeykt/Nuked-SC55@f6e5022](https://github.com/nukeykt/Nuked-SC55/commit/f6e50220d5de4241080e1811a30d29f7113a0929)
> 
> its now GPL.
> 
> good job releasing 0.83.0 btw. im still working on updating my configs

What hype? :) links? 

We'll keep using the CLAP plugin approach anyway. No point rewriting it and DREAMM and 86box both use my plugin, too.
